code/game/objects/structures/statues.dm

```diff
@@ -18,6 +18,22 @@
 	. = ..()
 	AddComponent(art_type, impressiveness)
 	INVOKE_ASYNC(src, /datum.proc/_AddComponent, list(/datum/component/beauty, impressiveness *  75))
+	AddComponent(/datum/component/simple_rotation,ROTATION_ALTCLICK | ROTATION_CLOCKWISE, CALLBACK(src, .proc/can_user_rotate),CALLBACK(src, .proc/can_be_rotated),null)
+
+/obj/structure/statue/proc/can_be_rotated(mob/user)
+	if(!anchored)
+		return TRUE
+
+/obj/structure/statue/proc/can_user_rotate(mob/user)
+	var/mob/living/L = user
+	if(istype(L))
+		if(!user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
+			return FALSE
+		else
+			return TRUE
+	else if(isobserver(user) && CONFIG_GET(flag/ghost_interaction))
```

Comment:
why should ghosts be able to rotate statues

Reply:
because I copypasted this from chairs :)